pkg/virt-controller/services/template.go

```diff
@@ -534,27 +580,26 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 				Add: capabilities,
 			},
 		},
-		Command:       command,
-		VolumeDevices: volumeDevices,
-		VolumeMounts:  volumeMounts,
-		ReadinessProbe: &k8sv1.Probe{
-			Handler: k8sv1.Handler{
-				Exec: &k8sv1.ExecAction{
-					Command: []string{
-						"cat",
-						"/tmp/healthy",
-					},
-				},
-			},
-			InitialDelaySeconds: int32(initialDelaySeconds),
-			PeriodSeconds:       int32(periodSeconds),
-			TimeoutSeconds:      int32(timeoutSeconds),
-			SuccessThreshold:    int32(successThreshold),
-			FailureThreshold:    int32(failureThreshold),
-		},
-		Resources: resources,
-		Ports:     ports,
+		Command:        command,
+		VolumeDevices:  volumeDevices,
+		VolumeMounts:   volumeMounts,
+		ReadinessProbe: defaultReadinessProbe,
+		Resources:      resources,
+		Ports:          ports,
+	}
+
+	if vmi.Spec.ReadinessProbe != nil {
+		container.ReadinessProbe = copyProbe(vmi.Spec.ReadinessProbe)
+		container.ReadinessProbe.InitialDelaySeconds = container.ReadinessProbe.InitialDelaySeconds + LibvirtStartupDelay
+	} else {
```

Comment:
you already set it, when you create container struct, so you can drop `else`

Reply:
ping @rmohr